### ISSUE TITLE
Bug when calling stop

### ISSUE THREAD

**Issue by benjsicam** [derickbailey/rabbus#11]
Getting error below when calling stop.

```
/path/to/project/node_modules/rabbus/rabbus/lib/consumer.js:34
  logger.info("Stopping Consumer For '" + this.options.queueOptions.name + "'");

TypeError: Cannot read property 'name' of undefined
    at Subscriber.Consumer.stop (/path/to/project/node_modules/rabbus/rabbus/lib/consumer.js:34:68)
```

**Comment by benjsicam** [comment]
I think it should just be:

``` javascript
logger.info("Stopping Consumer For '" + this.options.queue.name + "'");
```

**Comment by mxriverlynn** [comment]
thanks for this, @benjsicam! i'm trying to get it pulled in, but having problems with my local rmq install at the moment. will get it done asap, though.

**Comment by mxriverlynn** [comment]
alright, got this done and released as v0.6.2
